_docs/concepts/policy-and-control/mixer-aspect-config.md

```diff
@@ -0,0 +1,154 @@
+  params:
+    redisServerUrl: redisHost:6379
+    minDeduplicationDuration: 2s
+```
+
+The following example shows how to use a [_selector_](./mixer-config.html#Selectors) to apply rate limits selectively.
+
+```yaml
+- selector: source.labels["app"]=="reviews" && source.labels["version"] == "v3"  
+  aspects:
+  - kind: quotas
+    params:
+      quotas:
+      - descriptorName: RequestCount
+        maxAmount: 5
+        expiration: 1s
+        labels:
+          label1: target.service
+```
+
+
+## Aspect associations 
+The steps outlined in the previous section apply to all of Mixer's aspects.
+Each aspect requires specific `desciptors` and `adapters`.
+The following table enumerates valid combinations of the `aspects`, the `descriptors` and the `adapters`.
+
+
+|Aspect   |Descriptors               |Adapters
+|-----------------------------------------------
+|[Quota enforcement]({{aspectConfig}}/quotas.html ) | [QuotaDescriptor]({{mixerConfig}}#istio.mixer.v1.config.descriptor.QuotaDescriptor) |  [memQuota]({{adapterConfig}}/memQuota.html), [redisQuota]({{adapterConfig}}/redisQuota.html) 
+|[Metrics collection]({{aspectConfig}}/metrics.html)| [MetricDescriptor]({{mixerConfig}}#metricdescriptor) |[prometheus]({{adapterConfig}}/prometheus.html),[statsd]({{adapterConfig}}/statsd.html)  
+|[Whitelist/Blacklist]({{aspectConfig}}/lists.html)| None |[genericListChecker]({{adapterConfig}}/genericListChecker.html),[ipListChecker]({{adapterConfig}}/ipListChecker.html)  
+|[Access logs]({{aspectConfig}}/accessLogs.html)|[LogEntryDescriptor]({{mixerConfig}}#logentrydescriptor)  |[stdioLogger]({{adapterConfig}}/stdioLogger.html)
+|[Application logs]({{aspectConfig}}/applicationLogs.html)|[LogEntryDescriptor]({{mixerConfig}}#logentrydescriptor)  |[stdioLogger]({{adapterConfig}}/stdioLogger.html)
+|[Deny Request]({{aspectConfig}}/denials.html)| None |[denyChecker]({{adapterConfig}}/denyChecker.html)
+
+Istio uses [`protobufs`](https://developers.google.com/protocol-buffers/) to define configuration schemas. The [Writing Configuration]({{home}}/docs/reference/writing-config.html) document explains how to express `proto` definitions as `yaml`.
+
+
+## Organization of configuration
+Aspect configuration applies to a `subject`. A `Subject` is a resource in a hierarchy.
+Typically `subject` is the fully qualified name of a service, namespace or a cluster. An aspect configuration may apply
+to the `subject` resource and its sub-resources.
+
+## Pushing configuration
+`istioctl` pushes configuration changes to the API server.
+As of the alpha release, the API server supports pushing only aspect rules. 
+
+A temporary workaround allows you to push `adapters.yml` and `descriptors.yml` as follows.
+1. Find the mixer pod
+    ```
+    kubectl get pods -l istio=mixer
+    NAME                           READY     STATUS    RESTARTS   AGE
+    istio-mixer-2657627433-3r0nn   1/1       Running   0          2d
+    ```
+2. Fetch adapters.yml from Mixer
+    ```
```

Comment:
Need to have only 3 spaces before this PRE block as well as the previous. Otherwise, when rendered in HTML there is an extra space in the output, making pasted command-lines not work rightroperly.